scripts/gen_sqlc_docker.sh

```diff
@@ -21,8 +21,11 @@ docker run \
 # Until https://github.com/kyleconroy/sqlc/issues/1334 is fixed, we need to
 # manually modify some types so LEFT JOIN queries compile properly.
 echo "Fixing LEFT JOIN issue..."
-sed -i.bak -E 's/FamKeyFamily([[:space:]])+int32/FamKeyFamily\1sql.NullInt32/g' tarodb/sqlite/assets.sql.go
-sed -i.bak -E 's/FamKeyIndex([[:space:]])+int32/FamKeyIndex\1sql.NullInt32/g' tarodb/sqlite/assets.sql.go
+sed -i.bak -E 's/FamKeyFamily([[:space:]])+int32/FamKeyFamily\1sql.NullInt32/g' tarodb/sqlc/assets.sql.go
```

Comment:
q: i remember having some problems with sed compatibility in linux/mac. Did we try this one in mac os?

Reply:
The compat fix w/ mac os here is: `-i.bank -E`.